data/ecosystems/d/dantenetwork.toml

```diff
@@ -0,0 +1,6 @@
+title = "dantenetwork"
+sub_ecosystems = []
+github_organizations = ["https://github.com/dantenetwork"]
```

Comment:
we have a process that finds all repos under orgs and maps those repos to the parent ecosystem. I see that this org has 60+ repos but only `canister_contracts` should belong to IC. Could you add only that repo to icp? (Instead of this whole ecosystem)